bin/templates/scripts/cordova/lib/spawn.js

```diff
@@ -27,6 +27,7 @@ var proc = require('child_process');
  * @param  {String} opt_cwd       Working directory for command
  * @param  {String} opt_verbosity Verbosity level for command stdout output, "verbose" by default
  * @return {Promise}              Promise either fullfilled or rejected with error code
+ * @deprecated Use `require('cordova-common').superspawn` instead.
```

Comment:
I'd be inclined to just remove this outright in the next major, but I guess technically we don't know if anything else used it so we should go through one major cycle with it marked deprecated 🙁

Reply:
I also would favor updating `spawn.js` to output a warning message that it is deprecated and will be removed in the future.

Reply:
I'm somewhat resistant to just `console.log` in this context. If we're assuming that someone is just going to `require()` this file, we can't assume other logging constructs to be in place. And logging directly to console may be more disturbing than helpful because it might break code that parses console output (which wouldn't be unreasonable with this code).

So I'd rather leave it as is, unless someone has a very specific suggestion on how to approach the issue.

Reply:
Got it. I would like to raise to finish raising another PR to add the deprecation message, for separate discussion, before we resolving this conversation.